low-level/sensors/impl/centos_7/disk_monitor.py

```diff
@@ -96,6 +97,7 @@ class DiskMonitor(SensorThread, InternalMsgQ):
     NODE_DISK_RESOURCE_TYPE = "node:os:disk"
     ENCLOSURE_DISK_RESOURCE_TYPE = "enclosure:fru:disk"
     SMARTCTL_PASSED_RESPONSE = "SMART overall-health self-assessment test result: PASSED"
+    UDISKS2_UNAVAILABLE = "org.freedesktop.UDisks2 was not provided"

```

Comment:
This is the error we get if udisks not installed ?

Reply:
yes.